### DISCUSSION TITLE
non-Lambertian surface option

### DISCUSSION THREAD

**Discussion by Srheft** [eradiate/eradiate#341] [General]
Hi Eradiate,

Is there currently an option to choose a non-Lambertian surface such that it would exhibit multiple scatterings at a given VZA? 

Thanks,
S.

**Comment by leroyvn** [comment]
Hi @Srheft, we also provide [an implementation of the RPV BRDF](https://eradiate.readthedocs.io/en/latest/rst/reference_api/generated/autosummary/eradiate.scenes.bsdfs.RPVBSDF.html). We have more BRDFs on our roadmap. All our BRDFs model reflection as single scattering events—even though they can, in practice, account for multiple scattering events happening inside the surface at the microscale.

If you want proper multiple scattering modelling, we support explicit 3D canopy geometry as well as part of the [`CanopyAtmosphereExperiment`](https://eradiate.readthedocs.io/en/latest/rst/reference_api/generated/autosummary/eradiate.experiments.CanopyAtmosphereExperiment.html).

I hope this answers your question. Are there specific models you have in mind, which you'd like to see added to our list?

**Comment by Srheft** [comment]
I see. Thank you for the info! at the moment, I don't have a specific model to register; my initial goal is to pass similar input parameters to Eradiate that are passed to MODTRAN to be able to compare the outputs.  Besides the surface type, MODTRAN intakes Aerosol Optical Depth at 550nm and their spectral dependence ["Angstrum"] coefficient which I hope to find similar options for those inputs in Eradiate.
